sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -956,6 +957,13 @@ def test_sample():
                            for k in range(n_features)])
         assert_array_almost_equal(gmm.means_, means_s, decimal=1)
 
+        # Check that sizes that are drawn match what is requested
+        assert_equal(X_s.shape, (n_samples, n_components))
+        for sample_size in [4, 101, 1004, 5051]:
```

Comment:
Is there a particular reason to try with a large number? Would `for sample_size in range(50)` suffice?